descriptors/ModuleDescriptor-template.json

```diff
@@ -283,6 +283,47 @@
         }
       ]
     },
+    {
+      "id": "scheduled-notice-storage",
+      "version": "1.0",
```

Comment:
I'd like to suggest that we mark this interface as experimental, by using `0.1`, given that it is likely to change

Reply:
Changed version to 0.1